__tests__/InterpolateColor.test.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { View, Button } from 'react-native';
+import { render, fireEvent } from '@testing-library/react-native';
+import Animated, {
+  interpolateColor,
+  useSharedValue,
+  useAnimatedStyle,
+  withTiming,
+} from '../src';
+
+describe('colors interpolation', () => {
+  it('interpolates rgb without gamma correction', () => {
+    const colors = ['#105060', '#609020'];
```

Comment:
Let's add a few more tests:
1. CSS colors (i.e. from `red` to `blue`)
2. Semi-transparent colors (i.e. alpha != 1)